cli/src/commands/util.rs

```diff
@@ -99,6 +101,45 @@ pub(crate) struct UtilMarkdownHelp {}
 #[derive(clap::Args, Clone, Debug)]
 pub(crate) struct UtilConfigSchemaArgs {}
 
+#[doc = r#"Execute an external command via jj
+
+This is useful for arbitrary aliases. For example, assume you have a script
+called "my-jj-script" in you $PATH and you would like to execute it as "jj
+my-script". You would add the following line to your configuration file to
+achieve that:
+
+```toml
+[aliases]
+my-script = ["util", "exec", "--", "my-jj-script"]
+#                            ^^^^
+# This makes sure that flags are passed to your script instead of parsed by jj.
+```
+
+If you don't want to have to manage your script as a separate file, you can
+even inline a script into your config file:
+
+```toml
+[aliases]
+my-inline-script = ["util", "exec", "--", "bash", "-c", """
+#!/bin/bash
+set -euo pipefail
+echo "Look Ma, everything in one file!"
+echo "args: $@"
+""", ""]
+#    ^^
+# This last empty string will become "$0" in bash, so your actual arguments
+# are all included in "$@" and start at "$1" as expected.
+```
+"#]
```

Comment:
Nit: use a normal doc comment and use the `verbatim_doc_comment` clap attribute.